Documentation/kubernetes-upgrade.md

```diff
@@ -47,18 +46,15 @@ The kubelet would then restart the pod, and the new image version would be used.
 
 ### Upgrading Remaining Master Node Components (High-Availability)
 
-The kube-controller-manager, kube-scheduler, and kube-podmaster are all also deployed as static pods in `/etc/kubernetes/manifests`. However, in high-availability deployments, the kube-podmaster is responsible for making sure only a single copy of the controller-manager and scheduler are running cluster-wide.
+The kube-controller-manager and kube-scheduler are also deployed as static pods in `/etc/kubernetes/manifests`. In high-availability deployments, kube-controller-manager and kube-scheduler must be used with the leader-elect flag and can be run on all master nodes
```

Comment:
Maybe we should just merge this section with the one above it `### Upgrading kube-apiserver and kube-proxy`

Seems like it just boils down to updating all the manifests on the master nodes.